dds_web/security/auth.py

```diff
@@ -69,21 +69,50 @@ def get_user_roles_common(user):
 
 
 def verify_token_no_data(token):
-    user, claims = __verify_general_token(token)
+    claims = __verify_general_token(token)
+    user = user_from_subject(claims.get("sub"))
     del claims
     gc.collect()
     return user
 
 
+def verify_invite_token_no_data(token):
```

Comment:
`verify_invite_token_no_data` -- just checking that I understand: this is refering to that it's checking that there is enough data in the invite token? I think a name change would be good in that case. Not a deal breaker though

Reply:
No, it's to indicate that it doesn't return any of the sensitive data inside the token.

Reply:
Ah ok! Could we change to `verify_invite_token_no_sensdata`?

Reply:
Done!